plugins/faustwp/readme.txt

```diff
@@ -1,18 +1,18 @@
-=== Faust ===
+=== Faust.js ===

```

Comment:
You may want to confirm this doesn't break anything in the `versionPlugin.js` script for CI/CD. I know some of those scripts depend on the plugin name/changelog name.

Reply:
Thanks @blakewilson. There are no issues after running `versionPlugin.js`. You can see the output of running `node scripts/versionPlugin.js`. Will close that test PR once this is merged.